### DISCUSSION TITLE
[Laravel 9.x] Call to undefined method Illuminate\Session\SymfonySessionDecorator::put()

### DISCUSSION THREAD

**Discussion by hebbet** [livewire/livewire#4558] [7. Bugs (archived)] [answered]
I am working on migrating one of my projets to Laravel 9.0.

When a form is submited with livewire (installed is latest version 2.10.0) i always get.

Call to undefined method Illuminate\Session\SymfonySessionDecorator::put()

I also tested it with laravel jetstream:
composer create-project --prefer-dist laravel/laravel example-app dev-master
cd example-app
php artisan --version (returns 9.x-dev)
composer require laravel/jetstream
php artisan jetstream:install livewire --teams
npm install && npm run dev
Register yourself and login. 
After that go to "profile" and change field "name" etc.

![image](https://user-images.githubusercontent.com/1423115/150496513-baac975b-7754-4159-a9f5-ba5c5ddb08c8.png)

**Comment by hebbet** [comment, marked as answer]
fixed in 2.10.1

**Reply by joshhanley** [reply]
@hebbet thanks for testing and reporting back! 🙂
